lib/include_utils.js

```diff
@@ -5,6 +5,8 @@
 
 'use strict';
 
+var g = require('strong-globalize')();
```

Comment:
Use `const` please, see https://loopback.io/doc/en/contrib/style-guide-es6.html#variable-declarations

There are other places in this pull request where you are using `var` instead of `const` in new code, it would be great if you could fix them too (but I am not going to insist on that).

Reply:
My intention to keep it consistent with existing code. I'm open to have separate PRs to use `const` over `var`.

Reply:
The convention I am pushing for is to start using `const` in new code we are writing, so that the future migration from `var` to `const` can touch less files.

Not a big deal though.